thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/datalayer/bao/jdbc/MergedAnomalyResultManagerImpl.java

```diff
@@ -257,6 +257,16 @@ public MergedAnomalyResultDTO findById(Long id) {
     return convertMergedAnomalyBean2DTO(list);
   }
 
+  @Override
+  public List<MergedAnomalyResultDTO> findByStartTimeInRangeAndDetectionConfigId(long startTime, long endTime,
+      long detectionConfigId) {
+    Predicate predicate =
+        Predicate.AND(Predicate.LT("startTime", endTime), Predicate.GT("endTime", startTime),
+            Predicate.EQ("detectionConfigId", detectionConfigId));
+    List<MergedAnomalyResultBean> list = genericPojoDao.get(predicate, MergedAnomalyResultBean.class);
```

Comment:
Can you directly call `findByPredicate` from here?

Reply:
Will update in the next commit. Thanks